addons/binding/org.openhab.binding.wizlighting/src/main/java/org/openhab/binding/wizlighting/internal/utils/NetworkUtils.java

```diff
@@ -0,0 +1,87 @@
+/**
```

Comment:
This class should not be part of a binding but rather be provided by the underlying framework: Eclipse SmartHome.

Please create a PR against https://github.com/eclipse/smarthome/blob/master/bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/net/NetUtil.java

To obtain a specific broadcast address we have a pending PR: https://github.com/eclipse/smarthome/pull/4848